virttest/utils_npiv.py

```diff
@@ -341,3 +341,38 @@ def restart_multipathd(mpath_dev="", expect_exist=False):
         if expect_exist:
             return True
     return False
+
+
+def prepare_multipath_conf(conf_path="/etc/multipath.conf", conf_content="",
+                           replace_existing=False, restart_multipath=True):
+    """
+    Prepae multipath conf file.
+
+    :param conf_path: Path to the conf file.
+    :param conf_content: Content of the conf file.
+    :param replace_existing: True means to replace exsiting conf file.
+    :param restart_multipathd: True means to restart multipathd.
+    :return: The content of original conf, can be used to recover env.
+    """
+    default_conf_content = ("defaults {\n\tuser_friendly_names yes"
+                            "\n\tfind_multipaths yes\n}")
+    old_conf_content = ""
+    if not conf_content:
+        conf_content = default_conf_content
```

Comment:
Better not change the variable value, which passed in as parameter

Reply:
ok, changed to `new_conf_content = conf_content if conf_content else default_conf_content`
thx